server/test/tuist_web/live/previews_live_test.exs

```diff
@@ -76,4 +76,135 @@ defmodule TuistWeb.PreviewsLiveTest do
     # Then
     assert has_element?(lv, "span", "AppOne")
   end
+
+  test "displays track column with None when track is not set", %{
+    conn: conn,
+    organization: organization,
+    project: project
+  } do
+    # Given
+    _preview =
+      AppBuildsFixtures.preview_fixture(
+        project: project,
+        display_name: "AppOne",
+        supported_platforms: [:ios]
+      )
+
+    # When
+    {:ok, lv, _html} = live(conn, ~p"/#{organization.account.name}/#{project.name}/previews")
+
+    # Then
+    assert has_element?(lv, "span", "None")
```

Comment:
I'd be a more specific here. Since the test is matching a "column" you can probably use a selector that matches a table column. I'd do the same with the rest of the tests.